app/controlplane/pkg/biz/integration.go

```diff
@@ -314,7 +314,7 @@ func (uc *IntegrationUseCase) Delete(ctx context.Context, orgID, integrationID s
 }
 
 // List attachments returns the list of attachments for a given organization and optionally workflow
-func (uc *IntegrationUseCase) ListAttachments(ctx context.Context, orgID, workflowID string) ([]*IntegrationAttachment, error) {
+func (uc *IntegrationUseCase) ListAttachments(ctx context.Context, orgID, workflowID string) ([]*IntegrationAndAttachment, error) {
```

Comment:
It would be nice to add some quick integration tests

Reply:
Added!